packages/block-library/src/site-logo/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import icon from './icon';
+import edit from './edit';
+
+const { name } = metadata;
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Site Logo' ),
+	description: __( 'Show a site logo' ),
+	icon,
+	getEditWrapperProps( attributes ) {
+		const { align, width } = attributes;
+		if ( 'left' === align || 'center' === align || 'right' === align ) {
+			return { 'data-align': align, 'data-resized': !! width };
+		}
+	},
+	edit,
```

Comment:
Can we use the "align" hook to support alignments instead of having the logic in the block itself?

Reply:
Maybe not that important for this block but can we try  using the lighterBlockDom flag too and the `Block.*` component?

Reply:
What is the `lighterBlockDom` flag?

Reply:
Here's an example of refactor to use that flag https://github.com/WordPress/gutenberg/pull/23062 
We still need to add some docs about it.

Reply:
Done